Data/Scripts/001_Technical/002_RubyUtilities.rb

```diff
@@ -50,6 +50,10 @@ def cut(bitmap, width)
     end
     return new_string
   end
+
+  def numeric?
+    return self.scan(/^[+-]?([0-9]+)(?:\.[0-9]+)?$/).count > 1
```

Comment:
`> 1`? Surely `scan` would only produce exactly 1 result if self was a number.

How about `return self[/^[+-]?([0-9]+)(?:\.[0-9]+)?$/] ? true : false` to simplify it a bit?

Reply:
Yeah, it's a typo there.